04-materials/02-anatomy-of-extensions.md

```diff
@@ -2,16 +2,20 @@
 
 :::{hint} Learning objectives
 * Understand the difference between extensions, plugins, and widgets
-* Learn how to get started with writing an extension
-* ...
+* Understand how to get started with writing an extension
+* Understand how to structure and navigate an extension project
+* Practice and get comfortable with the development & testing loop
 :::
 
 :::{important} Outcomes
-After completing this module, you will have created a new extension from the
-[official template](https://github.com/jupyterlab/extension-template) and implemented a
-new widget for displaying a random image and caption from a curated set.
-The widget will be launchable from the {term}`command palette <command palette>` and the
-{term}`launcher <launcher>`.
+In this module, we will create a new extension starting from the
+[official template](https://github.com/jupyterlab/extension-template).
+We'll implement a new widget for displaying a random image and caption from a curated
+set.
+We'll implement launching this widget from the {term}`command palette <command palette>`
+and the {term}`launcher <launcher>`.
+We'll implement a user interaction which enables users to request a new random item in
+real time.
```

Comment:
Do we want this to still be a bullet point list?

Reply:
Love that.